src/integration-test/kotlin/integration/VacuumIntegrationTest.kt

```diff
@@ -0,0 +1,135 @@
+package integration
+
+import com.github.kittinunf.result.Result
+import com.github.kittinunf.result.failure
+import com.github.kittinunf.result.map
+import com.google.protobuf.InvalidProtocolBufferException
+import config.EthereumPasswords
+import config.TestConfig
+import config.loadConfigs
+import io.grpc.ManagedChannelBuilder
+import iroha.protocol.Queries
+import iroha.protocol.QueryServiceGrpc
+import jp.co.soramitsu.iroha.ModelProtoQuery
+import jp.co.soramitsu.iroha.ModelQueryBuilder
+import kotlinx.coroutines.experimental.async
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.fail
+import sidechain.eth.util.DeployHelper
+import sidechain.iroha.IrohaInitialization
+import sidechain.iroha.consumer.IrohaNetworkImpl
+import sidechain.iroha.util.ModelUtil
+import sidechain.iroha.util.getRelays
+import sidechain.iroha.util.toByteArray
+import java.math.BigInteger
+
+/**
+ * Integration tests for vacuum usecase.
+ */
+class VacuumIntegrationTest {
+
+    init {
+        IrohaInitialization.loadIrohaLibrary()
+            .failure {
+                println(it)
+                System.exit(1)
+            }
+    }
+
+    /** Configurations for tests */
+    private val testConfig = loadConfigs("test", TestConfig::class.java)
+
+    /** Ethereum password configs */
+    private val passwordConfig = loadConfigs("test", EthereumPasswords::class.java, "/ethereum_password.properties")
+
+    /** Ethereum utils */
+    private val deployHelper = DeployHelper(testConfig.ethereum, passwordConfig)
+
+    /** Notary account in Iroha */
+    val notaryAccount = testConfig.notaryIrohaAccount
+
+    /** Iroha keypair */
+    val keypair = ModelUtil.loadKeypair(testConfig.iroha.pubkeyPath, testConfig.iroha.privkeyPath).get()
+
+    /** Iroha network */
+    val irohaNetwork = IrohaNetworkImpl(testConfig.iroha.hostname, testConfig.iroha.port)
+
+    /** Iroha transaction creator */
+    val creator = testConfig.iroha.creator
+
+
+    @Test
+    fun testVacuum() {
+        val masterAccount = testConfig.notaryIrohaAccount
```

Comment:
It is better to create another IntegrationHelper class that will hold different functions. But it is the scope of another PR.

Never the less, here should be:
1) deploy free relays smart contracts
2) register users with these relays
3) transfer some money to these relays
4) call vacuum to transfer to master wallet
5) ensure everything is OK